sync/src/test/java/tech/pegasys/artemis/sync/PeerSyncTest.java

```diff
@@ -161,6 +161,8 @@ void sync_stoppedBeforeBlockImport() {
     when(peer.requestBlocksByRange(any(), any(), any(), any(), any())).thenReturn(requestFuture);
 
     final SafeFuture<PeerSyncResult> syncFuture = peerSync.sync(peer);
+    SyncStatus syncStatus = peerSync.getSyncStatus();
+    assertThat(syncStatus.highest_slot).isGreaterThan(UnsignedLong.valueOf(0));
```

Comment:
We should be able to assert the exact highest slot as it should be the slot of the remote peer we set up.